aspnet/fundamentals/configuration/sample/src/CustomConfigurationSource/ConfigurationContext.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using Microsoft.EntityFrameworkCore;
 
-namespace CustomConfigurationProvider
+namespace CustomConfigurationSource
```

Comment:
Not a fan of this change. At the type level an `IConfigurationSource` is something very specific that can be used to describes the parameters to construct an `IConfigurationProvider` and then construct it. 

At the conceptual level you have a database that that is the source of some configuration settings and you are writing a configuration provider to be able to extract the data from that source. So I would still call the whole thing a provider.